### PR TITLE
PLAT-124526: Fixed draggable TemperatureControl even though it is disabled

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Issue: if TemperatureControl had the disabled attribute, it could still be draggable

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Solution: Add a 'disabled' prop and pass it to the inner component ArcSlider

### Links
[//]: # (Related issues, references)
PLAT-124526

### Comments

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by ybsung — APPROVED** [review]
LGTM

File: samples/sampler/stories/default/TemperatureControl.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ storiesOf('Agate', module)
 		'TemperatureControl',
 		() => (
 			<TemperatureControl
+				disabled={boolean('disabled', Config)}
```
Comment (MikyungKim) [inline samples/sampler/stories/default/TemperatureControl.js:21]: Let's add this prop in `ArcSlider` story too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/LabeledIconButton` max-width so that huge sized icon will not be cut off
 - `agate/Spinner` to pause the animation when `paused` prop is true
 - `agate/Picker` picker item width in horizontal for silicon
+- `agate/TemperatureControl` so it isn't draggable when is disabled
```
Comment (MikyungKim) [inline CHANGELOG.md:14]:
```suggestion
- `agate/TemperatureControl` to not draggable when it's disabled
```

File: samples/sampler/stories/default/TemperatureControl.js (hunk 1 above)
Comment (daniel-stoian-lgp) [reply samples/sampler/stories/default/TemperatureControl.js:21]: ok

File: samples/sampler/stories/default/ArcSlider.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
-import {select, number} from '@enact/storybook-utils/addons/knobs';
+import {select, number, boolean} from '@enact/storybook-utils/addons/knobs';
```
Comment (MikyungKim) [inline samples/sampler/stories/default/ArcSlider.js:3]:
```suggestion
import {boolean, select, number} from '@enact/storybook-utils/addons/knobs';
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**